### PR TITLE
Support stream inheritance for the forked process, fixes #71

### LINKED ISSUE #71: Support stream inheritance for the forked process
Related to a [jline issue](https://github.com/jline/jline3/issues/77).
Java 7 introduced the ability to inherit standard streams using the [`inheritIO()`](https://docs.oracle.com/javase/7/docs/api/java/lang/ProcessBuilder.html#inheritIO()) method.
It would be really nice to be able to support it.

### PR DESCRIPTION
**Pull request by gnodet**

### CODE REVIEW

**Comment by gnodet** [comment]
@olamy Could you have another look please ?

File: src/main/java/org/codehaus/mojo/exec/ExecMojo.java [hunk 1]
```diff
@@ -280,7 +290,7 @@ else if ( !StringUtils.isEmpty( argsProp ) )
 
             commandLine.addArguments( args, false );
 
-            Executor exec = new DefaultExecutor();
+            Executor exec = inheritIo ? new ExtendedExecutor(inheritIo) : new DefaultExecutor();
```
Comment (rfscholte) [inline src/main/java/org/codehaus/mojo/exec/ExecMojo.java]: I don't understand the constructor; This will always be `new ExtendedExecutor(true)`

Comment (gnodet) [reply src/main/java/org/codehaus/mojo/exec/ExecMojo.java]: That was a way to minimize code changes, but a better fix would be
```
Executor exec = new ExtendedExecutor(inheritIo);
```

**Comment by gnodet** [comment]
Superseded by #196

**Pull request closed without merging**